src/main/webapp/app/guided-tour/guided-tour.component.html

```diff
@@ -22,15 +22,20 @@ <h3 jhiTranslate="{{ currentTourStep.headlineTranslateKey }}" class="headline" *
             <div class="tour-block__content">
                 <h5 jhiTranslate="{{ currentTourStep.subHeadlineTranslateKey }}" class="subHeadline" *ngIf="currentTourStep.subHeadlineTranslateKey"></h5>
                 <span jhiTranslate="{{ currentTourStep.contentTranslateKey }}"></span>
-                <div *ngIf="currentTourStep.externalUrl && currentTourStep.externalUrlTranslateKey">
-                    <a jhiTranslate="{{ currentTourStep.externalUrlTranslateKey }}" [href]="currentTourStep.externalUrl" target="_blank"></a>
-                </div>
-                <div *ngIf="currentTourStep.contentType === ContentType.IMAGE && currentTourStep.imageUrl">
+                <div *ngIf="currentTourStep.imageUrl">
                     <img src="{{ currentTourStep.imageUrl }}" />
                 </div>
-                <div *ngIf="currentTourStep.contentType === ContentType.VIDEO && currentTourStep.videoUrl">
+                <div *ngIf="currentTourStep.videoUrl">
                     <iframe [src]="sanitizer.bypassSecurityTrustResourceUrl(currentTourStep.videoUrl)" frameborder="0" allowfullscreen></iframe>
                 </div>
+                <div *ngIf="currentTourStep.externalUrl">
+                    <a
+                        [ngClass]="{ 'btn btn-primary': currentTourStep.linkType === LinkType.BUTTON }"
+                        jhiTranslate="{{ currentTourStep.externalUrlTranslateKey }}"
+                        [href]="currentTourStep.externalUrl"
+                        target="_blank"
```

Comment:
This introduces a potential security risk for the user: https://medium.com/@ali.dev/how-to-fix-target-blank-a-security-and-performance-issue-in-web-pages-2118eba1ce2f

Reply:
@thilo-behnke Oh I didn't know that. Thanks for the input!